doc/client_lua_api.md

```diff
@@ -845,13 +845,20 @@ Please do not try to access the reference until the camera is initialized, other
 
 ```lua
      {
-         x = number,
-         y = number,
-         max = number,
-         actual = number
+         x = number,       -- Horizontal FOV
+         y = number,       -- Vertical FOV
+         max = number,     -- The maximum of x and y
+         actual = number,  -- What the FOV of the player would be (not adjusted for zooming)
+         zoom = number     -- What the FOV of the player when zooming would be
```

Comment:
>_< I just pushed 5 seconds ago.
Should I make yet another commit?
If I do, maybe you should cancel the travis builds.